File: examples/elixir/basic-server/lib/basic_service_impl.ex
defmodule BasicServiceImpl do
  @behaviour BasicServiceBehaviour

  alias BasicServiceTypes

  @impl true
  def handle_get_type_closure(_ctx, _input, _meta), do: {:error, :not_implemented}

  @impl true
  def handle_list_basic_items(_ctx, _input, _meta), do: {:error, :not_implemented}
end

File: examples/erlang/basic-server/src/basic_service_impl.erl
-module(basic_service_impl).
-behaviour(basic_service_behaviour).

-include("basic_service_types.hrl").

-export([handle_get_type_closure/3, handle_list_basic_items/3]).

handle_get_type_closure(_Ctx, _Input, _Meta) ->
    {error, not_implemented}.

handle_list_basic_items(_Ctx, _Input, _Meta) ->
    {error, not_implemented}.
